### PR TITLE
Add FlyingChairs dataset

### PR DESCRIPTION
**Pull request by adriaangraas**
I've drafted up a FlyingChairs dataset, see #97 .

The dataset comes with the notice "Any commercial use is prohibited." and a very short Terms of Use but other than that does not provide a license. See [their webpage](https://lmb.informatik.uni-freiburg.de/resources/datasets/FlyingChairs.en.html). How should we inform users of that?

# Add Dataset

* Dataset Name: FlyingChairs
* Issue Reference: #97 
* `dataset_info.json` Gist: [here](https://gist.github.com/adriaangraas/9c8e9c2d22b8386616204ee450d7eb10)
  
## Description
The Flying Chairs are a synthetic dataset with optical flow ground truth. It consists of 22872 image pairs and corresponding flow fields. Images show renderings of 3D chair models moving in front of random backgrounds from Flickr. Motions of both the chairs and the background are purely planar. This dataset has been used for training convolutional networks in the ICCV 2015 paper FlowNet: Learning Optical Flow with Convolutional Networks.
  
## Checklist
* [x] Address all TODO's
* [x] Add alphabetized import to subdirectory's `__init__.py`
* [x] Run `download_and_prepare` successfully
* [x] Add checksums file
* [x] Properly cite in `BibTeX` format
* [x] Add passing test(s)
* [x] Add test data
* [x] Add data generation script (if applicable)
* [x] Lint code

### CODE REVIEW

**Comment by adriaangraas** [comment]
Thanks @ChanchalKumarMaji for reviewing. The images there are just the first few images from the dataset, I cannot generate these images myself. What would you suggest?

**Comment by cyfra** [comment]
You can generate fake images (see testing/fake_data_utils.py for some methods that generate fake jpeg or png images).

**Comment by adriaangraas** [comment]
I now generated random images and flow fields instead. I also added the generation script.

**Comment by ChanchalKumarMaji** [comment]
Can u please reduce the size of the fake data.

**Comment by adriaangraas** [comment]
@ChanchalKumarMaji less samples?

**Comment by adriaangraas** [comment]
@ChanchalKumarMaji what do you mean with "reduce the size"? Do you mean less samples? I have 5 samples, 3 in the train and 2 in the validation set.

**Comment by innat** [comment]
Any update on this PR?
